### PR TITLE
Added support for 3 dotnet-format arguments

### PR DESCRIPTION
**Pull request by bamancio-futura**
I have added support for the dotnet-format arguments `--fix-whitespace`, `--fix-analyzers-level`, and `--fix-style-level`.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - master
+      - main
```
Comment (bamancio-futura) [inline .github/workflows/ci.yml:7]: Taking into consideration the trend to use `main` instead of `master` should I rename this back to master?

Comment (jfversluis) [reply .github/workflows/ci.yml:7]: Nope this is perfect, I missed this one. Thanks!

**Comment by bamancio-futura** [comment]
> Awesome! Thanks for adding this! Did you try this out yourself somewhere?

Hey, nice you like it. Yeah, I currently use the action from my fork in a private repository, so it's running as expected. As soon as you merge it I'll switch back to your repo.

**Comment by jfversluis** [comment]
Perfect, let's do it!

**Pull request merged**